server/data/deliusClient.ts

```diff
@@ -79,6 +42,10 @@ module.exports = (signInService) => {
 
       return result.body
     } catch (error) {
+      if (error.status === 404) {
+        logger.debug('Returned 404', path, error.response, error.stack)
+        return undefined
```

Comment:
is there a reason to return undefined rather than null as we do elsewhere in here? 

I don't think debug logging is captured at all in application insights, might be nicer to have a simple info trace, following the pattern as below though possibly with the error stack. 

I think we need to add some context. - "we are calling delius" as the logging isn't name spaced as in java apps
The error.response object is massive which is why we just get the text of the response out below.

Reply:
I think I made this debug level because not found is a normal flow and nothing unusual.  As for undefined vs null - I'll change it.